### ISSUE TITLE
Resetting the Solace Admin Password is causing the passive pods to fail

### ISSUE THREAD

**Issue by mrvivektiwari** [SolaceProducts/pubsubplus-kubernetes-helm-quickstart#115]
Description:-
  Background:-
  we have to rotate our solace admin password  in our production environment regularly so we need a better strategy to reset our 
  password regularly with no downtime in solace.

  we try to reset the password via both CLI and solace web console but ended up with the same result.

Issue Detail:-
 we have solace cluster running on 3 pods with redundancy on. We try to reset the password by logging into the solace web 
 console.
 once we hit the apply button with the new password it forcefully logout us and then the page is not accessible.
 The pods are failing to be in a ready state due to the liveness and readiness probes failing.
 `<returnInfo><errorInfo>management host is not responding</errorInfo></returnInfo>
  INFO: startup-broker.sh-Waited 170 seconds, Management API not yet accessible`

looking for a fix for this.

**Comment by bczoma** [comment]
Hi @vivekcs0114 , the reason for this is that the broker is still using internally the outdated originally configured password to access the Management API. The secret belonging to the deployment is the one storing the outdated password. You'll need to update the secret to the new password immediately before making the password update admin change. A subsequent pod restart will mount the modified secret with the new password. Please ensure to try this in a test environment first.

**Comment by bczoma** [comment]
I just came across this and found that there is no need for pod restarts. The changed secret is mounted automatically for all pods. However two CLI pw change steps are necessary: change admin pw on active broker, this will update the standby. Additionally it is still required to update the admin password on the monitor via direct CLI there.

**Comment by shukla2009** [comment]
Isn't this is race condition? as after modifying the password in helm/secret which is updated in pod is outdated and can't be used till updated via cli/console so any restart in between will lead to outage ? 

1) Can we create a less privileged user to be used in scripts and let admin user to be initialised only once ? 
2) Is  ldap an option ?

**Comment by bczoma** [comment]
[User management considerations](https://github.com/SolaceProducts/pubsubplus-kubernetes-helm-quickstart/blob/master/docs/PubSubPlusK8SDeployment.md#user-management-considerations) section has now been added to the documentation.

The solution of a less privileged user to be used in scripts is being considered for the Operator-based deployment.

Ldap is an option for user management, however it needs to be configured after the broker deployment so it is out of scope for this quickstart.

**Comment by bczoma** [comment]
Closing this issue.
